airflow/api/common/experimental/dag_runs.py

```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""DAG runs APIs."""
+from typing import Optional, List, Dict, Any
+from flask import url_for
+
+from airflow.api.common.experimental import check_and_get_dag
+from airflow.models import DagRun
+from airflow.utils import timezone
+
+
+def get_dag_runs(dag_id: str, state: Optional[str] = None, state_ne: Optional[str] = None,
+                 execution_date_before: Optional[str] = None, execution_date_after: Optional[str] = None,
+                 execution_date: Optional[str] = None) -> List[Dict[str, Any]]:
+    """
+    Returns a list of Dag Runs for a specific DAG ID.
+
+    :param dag_id: String identifier of a DAG
+    :param state: queued|running|success...
+    :param state_ne: queued|running|success...
+    :param execution_date_before: a query string parameter to find all runs before provided date,
+    should be in format "YYYY-mm-DDTHH:MM:SS", for example: "2016-11-16T11:34:15".
```

Comment:
```suggestion
        should be in format "YYYY-mm-DDTHH:MM:SS", for example: "2016-11-16T11:34:15".
```
Ident is required.

Reply:
fixed.

Reply:
looking for identical problems elsewhere as well. so not - 'fixed'

Reply:
fixed as many as I could find.